mtgjson5/__main__.py

```diff
@@ -102,6 +103,13 @@ def dispatcher(args: argparse.Namespace) -> None:
         return
 
     sets_to_build = ScryfallProvider().get_sets_to_build(args)
+    if args.all_sets:
+        with constants.RESOURCE_PATH.joinpath("additional_sets.json").open(
```

Comment:
This code is used in two places, can we abstract it out?

Reply:
I thought about building a provider but we don't use providers for any other resource file.

Reply:
I don't think we need a provider, more so a common util method to get access to this data

Reply:
Reworked to use util function